apps/portal/src/content/docs/components/alert-dialog.mdx

```diff
@@ -0,0 +1,341 @@
+---
+title: AlertDialog
+description: AlertDialog component documentation
+---
+
+The `AlertDialog` component provides a flexible UI element for displaying alert dialogs. It is composed of several subcomponents such as `AlertDialogHeader`, `AlertDialogFooter`, `AlertDialogTitle`, and `AlertDialogDescription` to offer a structured and customizable interface.
+
+import { DocsPage } from "@/components/docs-page";
+import { AlertDialog } from "@harnessio/ui/components";
+import { Button } from "@harnessio/ui/components";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<AlertDialog.Root>
+    <AlertDialog.Trigger asChild>
+      <Button>Open Alert Dialog</Button>
+    </AlertDialog.Trigger>
+    <AlertDialog.Content>
+      <AlertDialog.Header>
+        <AlertDialog.Title>Alert Dialog Title</AlertDialog.Title>
+      </AlertDialog.Header>
+      <AlertDialog.Description variant="secondary">
+        <p>Alert dialog content goes here. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
```

Comment:
The `AlertDialog.Description` component wraps the content in a `p` tag already, so this triggers warnings in the console for being a paragraph inside a paragraph.
```suggestion
      <AlertDialog.Description variant="secondary">
        Alert dialog content goes here. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```